README.md

```
@@ -38,19 +40,39 @@ class HomepageTest < ApplicationSystemTestCase
+# Step 2: Commit baselines to git
 git add doc/screenshots/
 git commit -m "chore: add screenshot baselines"
-bundle exec rake test                    # Second run compares against committed baselines
+
+# Step 3: Now comparisons work — change your UI and re-run
+bundle exec rake test
 ```
 
-First run saves baseline screenshots to `doc/screenshots/` (always passes). Commit them to git. Subsequent runs compare against committed baselines — if the UI changed, the test fails.
+After Step 1, you'll see:
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)